### PR TITLE
Add instruct-pix2pix notebook to model garden

### PR DESCRIPTION
**Pull request by xiangxu-google**
**REQUIRED:** Add instruct-pix2pix notebook to model garden

**REQUIRED:** Fill out the below checklists or remove if irrelevant
2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [y ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [y ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb [hunk 1]
```diff
@@ -0,0 +1,454 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: <p>Move H1 heading to first line in next links cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: <p>Vertex-AI -&gt; Vertex AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: <p>Vertex-AI API -&gt; Vertex AI SDK</p><p> </p><p>Make this a heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: <p>I suggest explaining the model is inside the container</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: <p>Add a heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: <p>delete the GCS bucket?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: Done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: Done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: Done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: I used a comment below to explain:
`# The pre-built serving docker image. It contains serving scripts and models.`

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: These sections belong to the same topic.
Moved explaining cells to the top of the section to make them less separated.

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb:1]: The GCS bucket stores experiments output which can be taken away by users.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**